### ISSUE TITLE
roadmap

### ISSUE THREAD

**Issue by gedw99** [wboayue/ibapi#8]
I would be happy to help on this as I us IB.

Their Desktop beta is Java and pretty awful.

https://github.com/g45t345rt/g45w might interest you as an example of how good we can get a gui also using golang.  I imagine you're not at all interested in a GUI, but only API ?

I am interested I both and happy to help on the API

**Comment by gedw99** [comment]
Seems this does streaming ( reading of data ) but no order transactions ?

**Comment by wboayue** [comment]
@gedw99 I agree that their desktop app can definitely be improved upon. However, the GUI isn't my focus. I started the Go API to support an algorithmic trading application that I've since migrated to Rust. I do, at some point, want to complete the order management API.

**Comment by gedw99** [comment]
Sh so no point in eating time in this repo since you moved to a rust repo ?
